code/modules/vending/wardrobes.dm

```diff
@@ -254,6 +287,11 @@
 					/obj/item/clothing/shoes/galoshes = 1,
 					/obj/item/watertank/janitor = 1,
 					/obj/item/storage/belt/janitor = 1)
+	contraband = list(/obj/item/storage/box/lights/tubes = 3,
+						/obj/item/storage/box/lights/bulbs = 3,
+						/obj/item/clothing/under/maid = 1)
+	premium = list(/obj/item/storage/box/lights/mixed = 2,
+					/obj/item/reagent_containers/spray/cleaner = 1)
```

Comment:
doesn't belong in a wardrobe but otherwise a great addition to the janitor vendor